ui/perfherder/sortHelpers.js

```diff
@@ -0,0 +1,63 @@
+export const tableSort = {
```

Comment:
I think it's better to move this file in a helper directory at the perfherder level, not treeherder, since we use the functionalities only in perfherder.